app/webpacker/components/wca/RegionFlag.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Popup } from 'semantic-ui-react';
+import classnames from 'classnames';
+import { countries } from '../../lib/wca-data.js.erb';
+
+/** Works with countries, continents, and world. */
+export default function RegionFlag({ iso2, withoutTooltip = false }) {
+  if (withoutTooltip) {
+    return <RegionFlagWithoutTooltip iso2={iso2} />;
+  }
+
+  return <RegionFlagWithTooltip iso2={iso2} />;
+}
+
+function RegionFlagWithTooltip({ iso2 }) {
+  return (
+    <Popup
+      id="resultCountryFlagTooltip"
+      position="top center"
+      content={countries.byIso2[iso2].name}
+      trigger={<span><RegionFlagWithoutTooltip iso2={iso2} /></span>}
```

Comment:
Why the extra `span`?

Reply:
For some reason, the `trigger` needs to explicitly start with a `span`, or it doesn't work. There's a comment about it in the `EventSelector` component. I cannot explain why.

Reply:
I added a comment, but there are a lot of instances in the codebase where we wrap a trigger in a span without any explanation.

Reply:
These are not our immediate concern. Probably a SemUI glitch that we can get rid of "for free" when moving frameworks.